packages/vsx-registry/src/browser/vsx-extensions-contribution.ts

```diff
@@ -143,6 +143,9 @@ export class VSXExtensionsContribution extends AbstractViewContribution<VSXExten
             commandId: VSXExtensionsCommands.INSTALL_ANOTHER_VERSION.id,
             label: nls.localizeByDefault('Install Another Version...'),
         });
+        menus.registerSubmenu(VSXExtensionsContextMenu.CONTRIBUTION, '', {
```

Comment:
The example menu is shown for me on an extension, but when I try to execute the action, I have a failure with rpc message. There are 2 arguments passed for me to the handler (position and VSXExtension selected). I think the latter one would need to be removed, or at least processed before being passed to RPC. 

Message:
```
Uncaught EncodingError Error: Error during encoding: 'Unknown type: symbol'
    at encode (/home/remi/Git/theia/packages/core/src/common/message-rpc/rpc-message-encoder.ts:151:23)
    at request (/home/remi/Git/theia/packages/core/src/common/message-rpc/rpc-message-encoder.ts:137:14)
    at sendRequest (/home/remi/Git/theia/packages/core/src/common/message-rpc/rpc-protocol.ts:161:22)
    at <anonymous> (/home/remi/Git/theia/packages/plugin-ext/src/common/proxy-handler.ts:76:45)
    --- await ---
    at delegate (/home/remi/Git/theia/packages/core/src/common/menu/menu-adapter.ts:69:43)
```

Reply:
Fixed and the command also correctly return the extension id as its first arg